### PR TITLE
Pm 866aagd70 add text records

### PR DESCRIPTION
**Pull request by makhnovskiy**

### CODE REVIEW

File: services/Route53Service.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    const input = {
+      HostedZoneId: Env.get('AWS_ETH_XYZ_HOSTED_ZONE_ID'),
+      StartRecordName: domain,
+      StartRecordType: 'TXT',
+      MaxItems: 10,
+    };
+    const command = new ListResourceRecordSetsCommand(input);
+
+    try {
+    const response = await client.send(command);
```
Comment (TonnyORG) [inline services/Route53Service.ts:34]: @makhnovskiy please fix indentation

File: services/Route53Service.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    const input = {
+      HostedZoneId: Env.get('AWS_ETH_XYZ_HOSTED_ZONE_ID'),
+      ChangeBatch: {
+        Changes: [
+          {
+            Action: action,
+            ResourceRecordSet: {
+              Name: txtKey,
+              Type: 'TXT',
+              TTL: 300,
```
Comment (TonnyORG) [inline services/Route53Service.ts:100]: @makhnovskiy increase this to 3600 (1 hour)

File: .env.example [hunk 3]
```diff
@@ -26,3 +26,8 @@ POKT_PORTAL_SECRET=
 OPENSEA_API_KEY=
 
 SENTRY_DSN=
+
+AWS_KEY=
+AWS_SECRET= 
+AWS_REGION=
+AWS_ETH_XYZ_HOSTED_ZONE_ID=
```
Comment (TonnyORG) [inline .env.example:33]: @makhnovskiy can you rename this to `AWS_HOSTED_ZONE_ID` instead? The reason is, the project is open source so anyone can take it a put it into their own domain so I would prefer to keep this as "brandless/standard" as  possible.

**Review by TonnyORG — CHANGES_REQUESTED** [review]
A few changes please

File: services/Route53Service.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  async setRecord(txtKey, txtValue, action = 'UPSERT') {
+    const client = new Route53Client(clientConfig);
+
+    // remove double quotes if so returned by the api
+    if(txtValue && txtValue[0] == '"' && txtValue[txtValue.length - 1] == '"'){
+      txtValue = txtValue.substring(1, txtValue.length-1);
+    }
+
+    const input = {
+      HostedZoneId: Env.get('AWS_ETH_XYZ_HOSTED_ZONE_ID'),
```
Comment (TonnyORG) [inline services/Route53Service.ts:92]: @makhnovskiy forgot to update this

**Review by TonnyORG — CHANGES_REQUESTED** [review]

File: services/Route53Service.ts (hunk 4 above)
Comment (makhnovskiy) [reply services/Route53Service.ts]: @TonnyORG My bad - GitHub Desktop didn't recognize the file and didn't push... Fixed now

**Review by TonnyORG — APPROVED** [review]

**Pull request merged**